recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "0.12.0" %}
+{% set version = "0.13.0rc4" %}
 {% set cov_fail_under = "82" %}
```

Comment:
```
Required test coverage of 82% reached. Total coverage: 85.28%
```

Let's crank the coverage baseline to 85

Reply:
No problem. I have increased it. But I don't think the coverage testing belongs here in this repo :)

Reply:
A bit of context: as a maintainer of several hundred feedstocks: running the test suite, under coverage, provides a first line of defense against optional (but important for whatever reason) dependency drift.

A package known to be under full test is much safer to have `automerge` enabled (as this one does, though I take issue with that #29). Fixing a few deps, and assessing changes when things fail is much less work, at scale, and crucially requires less total intervention than waiting for an end user to have a bad day, maybe complaining to the upstream, finally landing here, making a patch release, maybe making a repodata patch, etc.